### DISCUSSION TITLE
Bread crumbs

### DISCUSSION THREAD

**Discussion by RaSabirov** [requarks/wiki#6278] [Help / Questions] [answered]
How to remove click on bread crumbs in code?)))
![image](https://user-images.githubusercontent.com/89607273/226268416-004266a1-b6c3-4f8a-83c4-d267411c994a.png)

**Comment by vladyxa13** [comment]
Ravil is is active page? Or this on hover?

**Reply by RaSabirov** [reply]
**on hover**

I want to remove hover and click on breadcrumbs. I can't find where it is in the code.
so i need help))

**Reply by vladyxa13** [reply]
![Screenshot_ripple-content](https://user-images.githubusercontent.com/61389369/226403387-300769f2-567a-41f8-8ddc-39d039baf235.png)
it occurs because this span have this effect, need to use style for this `span` or google info about this Vue.js element and insert patch into Admin-Theme-CSS

**Comment by RaSabirov** [comment, marked as answer]
Done.

Embedding in HTML <head> in Admin -> Theme -> CSS: 
<style>.v-breadcrumbs{pointer-events: none}</style>
